dhis-2/dhis-api/src/main/java/org/hisp/dhis/dataintegrity/DataIntegrityCheckType.java

```diff
@@ -92,7 +92,7 @@
     PROGRAM_RULE_ACTIONS_WITHOUT_DATA_OBJECT,
     PROGRAM_RULE_ACTIONS_WITHOUT_NOTIFICATION,
     PROGRAM_RULE_ACTIONS_WITHOUT_SECTION,
-    PROGRAM_RULE_ACTIONS_WITHOUT_STAGE;
+    PROGRAM_RULE_ACTIONS_WITHOUT_STAGE_ID;
```

Comment:
Had to rename this to make it's code unique. I did clash with the one above. We will remove these later anyhow so this is just to make the uniqueness tests pass until these get removed.